src/ViewRenderer/ViewRenderer.php

```diff
@@ -1,48 +1,44 @@
 <?php
 
-namespace App;
+declare(strict_types=1);
+
+namespace App\ViewRenderer;
 
 use Psr\Http\Message\ResponseInterface;
 use Yiisoft\Aliases\Aliases;
-use Yiisoft\Router\UrlMatcherInterface;
+use Yiisoft\Arrays\ArrayHelper;
 use Yiisoft\Strings\Inflector;
 use Yiisoft\View\ViewContextInterface;
 use Yiisoft\View\WebView;
 use Yiisoft\DataResponse\DataResponseFactoryInterface;
-use Yiisoft\Yii\Web\Middleware\Csrf;
-use Yiisoft\Yii\Web\User\User;
 
 final class ViewRenderer implements ViewContextInterface
```

Comment:
Can part of these changes such as having defaults for parameters / layout be applied to yiisoft/view package directly?

Reply:
Also do you plan to move this class somewhere out of the application?

Reply:
> Also do you plan to move this class somewhere out of the application?

https://github.com/yiisoft/yii-view

Reply:
> Can part of these changes such as having defaults for parameters / layout be applied to yiisoft/view package directly?

This can be partially done. For link and meta tags.
For content/layout params this it's impossible: in yiisoft/view there is no division into content and layout.

Reply:
Let's add default link and meta-tags to yiisoft/view.